app/plot_app/statistics_plots.py

```diff
@@ -469,7 +468,7 @@ def _plot_public_data_statistics(self, all_data, version_attr_name,
                 if not d in version_type_data:
                     version_type_data[d] = 0.
                 data_hours[d].append(version_type_data[d])
-
+        print("Version!!",versions,self._version_data)
```

Comment:
Same here and below

Reply:
same as here is this the outdated code?

Reply:
Hmm... are you meaning that remove these line in that specific commit?

Reply:
removed from history.